modules/ROOT/pages/functions/dev_guide/nodejs/invoking-nodejs.adoc

```diff
@@ -1,11 +1,45 @@
 // Module included in the following assemblies
 // /functions/user_guide/develop-nodejs.adoc
 
-// [id="invoking-nodejs_{context}"]
-// = Invoking Node.js functions
+[id="invoking-nodejs"]
+= Invoking Node.js functions
 
 Functions can be invoked with a simple HTTP request. When an incoming request is received, functions are invoked with a `context` object as the first parameter.
 
+== Context objects
+
+Functions are invoked by calling a `context` object as the first parameter.
+
+.Example context object
+[source,js]
+----
+function processPurchase(context, data)
+----
+
+This object provides access to the incoming HTTP request information, including the HTTP request method, any query strings or headers sent with the request, the HTTP version, and the request body. If the incoming request is a CloudEvent, the CloudEvent itself will also be found on the context object.
+// this isn't clear what it means to me - isn't it all HTTP? Do we mean the CE data?
```

Comment:
Yes, all incoming requests are HTTP. Incoming requests that contain a [CloudEvent](https://cloudevents.github.io/sdk-javascript/classes/cloudevent.html) will attach the incoming instance of the `CloudEvent` to the `context` object so that it can be accessed using `context.cloudevent`. Does that help?